corehq/apps/app_manager/views/download.py

```diff
@@ -289,21 +290,8 @@ def resolve_path(path):
                     pass
                 else:
                     # this resource should exist but doesn't
-                    logging.error(
-                        'Expected build resource %s not found' % path,
-                        extra={'request': request}
-                    )
-                    if not request.app.build_broken:
-                        request.app.build_broken = True
-                        request.app.build_broken_reason = 'incomplete-build'
-                        try:
-                            request.app.save()
-                        except ResourceConflict:
-                            # this really isn't a big deal:
-                            # It'll get updated next time a resource is request'd;
-                            # in fact the conflict is almost certainly from
-                            # another thread doing this exact update
-                            pass
+                    _assert = soft_assert('@'.join(['jschweers', 'dimagi.com']))
+                    _assert(False, 'Expected build resource %s not found' % path)
```

Comment:
@benrudolph yeah given the background this ticket, I agree with @orangejenny. Basically I believe this check is no longer relevant ever since we started making builds more atomically years ago, and we found that it was needlessly flagging builds as broken during a riak outage.